tools/clang/test/CodeGenSPIRV_Lit/shader.debug.function.hlsl

```diff
@@ -1,33 +1,33 @@
-// RUN: %dxc -T ps_6_0 -E main -fspv-debug=vulkan
+// RUN: %dxc -E main -T ps_6_0 -spirv -fcgl -Vd -fspv-debug=vulkan -Qembed_debug  %s | FileCheck %s
```

Comment:
validation wasn't disabled before, is that expected?
Is there an issue pending for this? Or is it a bad test?

Reply:
Good catch.

The reason I added -Vd here is to match 
// CHECK:         [[clOpts:%\d+]] = OpString " -E main -T ps_6_0 -spirv -fcgl -Vd -fspv-debug=vulkan -Qembed_debug" 

If it is OK to remove the -Vd in this CHECK line, we can remove the -Vd in the RUN line.

Do you prefer remove -Vd?

Reply:
I believe removing `-Vd` from both would be best. Same for `Qembed_debug`, don't think that's required for SPIR-V debug informations since we don't emit PDB.

Reply:
Cool.
I'll remove -Vd and Qembed_debug.

Reply:
The output still have -Qembed_debug,
Changed to
OpString " -E main -T ps_6_0 -spirv -fcgl -fspv-debug=vulkan